test/testdefs.jl

```diff
@@ -1,7 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 function runtests(name, isolate=true)
-    Base.Test.DISABLE_TESTSET_PRINTING[] = true
+    Base.Test.testset_print_enable(false)
```

Comment:
Unnecessary if it's called in exactly one place and not intended to be ever touched by anyone else.